node-notes-cli/app.js

```diff
@@ -0,0 +1,40 @@
+const fs = require('fs');
+const data = require('./data.json');
+const input = process.argv.filter((x, i) => i > 1);
```

Comment:
It's better to use `Array.prototype.slice()`. `Array.prototype.filter` goes through every element of the array, but you already know you aren't interested in anything before index `2`.